samples/bot-sharepoint-file-viewer/nodejs/README.md

```diff
@@ -0,0 +1,87 @@
+Using this Nodejs sample, a bot can upload file to SharePoint site and same files can be viewed in Teams file viewer.
```

Comment:
a bot with capability to upload files to SharePoint site and same files can be viewed in Teams file viewer